lm_eval/__main__.py

```diff
@@ -373,7 +382,7 @@ def cli_evaluate(args: Union[argparse.Namespace, None] = None) -> None:
 
         args.model_args = args.model_args + ",trust_remote_code=True"
 
-    eval_logger.info(f"Selected Tasks: {task_names}")
+    eval_logger.warning(f"Selected Tasks: {task_names}")
```

Comment:
Maybe this should just be a print? I get the desire to have it be higher-log level but WARNING seems like users will expect something is going wrong.